File: backend/src/main/java/com/smartcampus/backend/dto/weather/HourlyAirQualityPointDTO.java
package com.smartcampus.backend.dto.weather;

import lombok.*;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class HourlyAirQualityPointDTO {
    private String time;
    private Double pm2_5;
    private Double pm10;
    private Double ozone;
    private Double nitrogenDioxide;
    private Integer europeanAqi;
    private String category;
    private String description;
    private String icon;
}

File: frontend/app/src/main/java/com/ObservatoireCampus/mobile/model/weather/HourlyAirQualityPointDto.kt
package com.ObservatoireCampus.mobile.model.weather

data class HourlyAirQualityPointDto(
    val time: String,
    val pm2_5: Double?,
    val pm10: Double?,
    val ozone: Double?,
    val nitrogenDioxide: Double?,
    val europeanAqi: Int?,
    val category: String?,
    val description: String?,
    val icon: String?
)